internal/wal_push_handler.go

```diff
@@ -23,18 +24,40 @@ func (err CantOverwriteWalFileError) Error() string {
 	return fmt.Sprintf(tracelog.GetErrorFormatter(), err.error)
 }
 
+func  GetWalOverwriteSetting() (preventWalOverwrite bool, err error) {
```

Comment:
Move it to configure.go and rename similar to other ConfigureSomething in this file.

Reply:
Done